From 25c478492b889fe857c7e1a3fa2c43ff956a36bb Mon Sep 17 00:00:00 2001
From: Harsimran Singh Tungal <harsimransingh.tungal@arm.com>
Date: Wed, 24 Sep 2025 13:42:25 +0000
Subject: [PATCH 1/2] corstone1000: dts: use 32-bit cells for /reserved-memory
 node

Switch the *reserved-memory* node from two-cell (64-bit) encoding to
one-cell (32-bit) encoding and adjust the `reg` property accordingly
to make reserved-memory node format compatible with rest of the dts.

Upstream-Status: Pending
Signed-off-by: Harsimran Singh Tungal <harsimransingh.tungal@arm.com>
---
 arch/arm/dts/corstone1000-u-boot.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/dts/corstone1000-u-boot.dtsi b/arch/arm/dts/corstone1000-u-boot.dtsi
index 023109fb6a7..92c493c5692 100644
--- a/arch/arm/dts/corstone1000-u-boot.dtsi
+++ b/arch/arm/dts/corstone1000-u-boot.dtsi
@@ -18,12 +18,12 @@
 	};
 
 	reserved-memory {
-		#address-cells = <2>;
-		#size-cells = <2>;
+		#address-cells = <1>;
+		#size-cells = <1>;
 		ranges;
 
 		smem_mem: smem_region@88000000 {
-			reg = <0x0 0x88000000 0x0 0x100000>;
+			reg = <0x88000000 0x100000>;
 			no-map;
 		};
 	};
-- 
2.43.0

